pkg/performanceprofile/cmd/render/render.go

```diff
@@ -222,6 +223,21 @@ func render(inputDir, outputDir string) error {
 			}
 			klog.Info(fileName)
 		}
+
+		//write PerformanceProfile to output
+		ppBin, err := yaml.Marshal(pp)
+		if err != nil {
+			return fmt.Errorf("unable to marshal PerformanceProfile %s. error: %w", pp.Name, err)
+		}
+		fileName := fmt.Sprintf("%s_pp.yaml", pp.Name)
```

Comment:
We should use the same name we got from the input. That will allow clean overwrite in the installer.

Reply:
yeah makes sense.
thanks for the comment